example/src/tests.ts

```diff
@@ -92,6 +92,33 @@ test("createFromKeyBundle throws error for non string value", async () => {
   return false;
 });
 
+test("canPrepareMessage", async () => {
+  const bob = await XMTP.Client.createRandom({ env: "local" });
+  const alice = await XMTP.Client.createRandom({ env: "local" });
+  await delayToPropogate();
+
+  const bobConversation = await bob.conversations.newConversation(
+      alice.address,
+  );
+  await delayToPropogate();
+
+  const prepared = await bobConversation.prepareMessage("hi");
+
+  // Either of these should work:
+  await bobConversation.sendPreparedMessage(prepared);
+  // await bob.sendPreparedMessage(prepared);
```

Comment:
This is the expected typical scenario for JS-only apps -- calling `.prepareMessage` immediately upon user input and then attempting `.sendPreparedMessage` (or waiting to (re)try when connectivity resumes).